README.md

```diff
@@ -121,13 +121,33 @@ gradlew build -x test
 
 **Note:** SpotBugs is not compatible with JDK 11+. SpotBugs will not be executed if you are using JDK 11+.
 
+### Importing into eclipse
+
+to import source project into eclipse
+
+```sh
+# for Linux/macOS:
+./gradlew eclipse
+
+
+# for Windows:
+gradlew eclipse
+
+```
+
+in eclipse 
+
+file->import->gradle->existing gradle project
+
+**Note:** please set your workspace text encoding setting to UTF-8
+
 ## Community
 
 Join our [<img src='https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_slack-512.png' width='20px' /> slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with the development team, for questions and discussions.
 
 Follow our [<img src='https://cdn2.iconfinder.com/data/icons/social-media-2285/512/1_Twitter_colored_svg-512.png' width='20px' /> twitter](https://twitter.com/deepjavalibrary) to see updates about new content, features, and releases.
 
-关注我们 [<img src='https://www.iconfinder.com/icons/5060515/download/svg/512' width='20px' /> 知乎专栏](https://zhuanlan.zhihu.com/c_1255493231133417472) 获取DJL最新的内容！
+å…³æ³¨æˆ‘ä»¬ [<img src='https://www.iconfinder.com/icons/5060515/download/svg/512' width='20px' /> çŸ¥ä¹Žä¸“æ �](https://zhuanlan.zhihu.com/c_1255493231133417472) èŽ·å�–DJLæœ€æ–°çš„å†…å®¹ï¼�
```

Comment:
It looks like those Chinese characters is trashed due to charset issue.
Please make sure your editor support UTF-8.
```suggestion
关注我们 [<img src='https://www.iconfinder.com/icons/5060515/download/svg/512' width='20px' /> 知乎专栏](https://zhuanlan.zhihu.com/c_1255493231133417472) 获取DJL最新的内容！
```